README.md

```diff
@@ -33,8 +34,9 @@ app.post(...)
 
 ### Options
 
-- `appName`                     - Name that will be used in the label for every metric
-- `ignoredRoutes` (optional)    - An array of routes to be exuded when calculating metrics. Default value: `['/metrics', '/healthz']`
+- `appName`                             - Name that will be used in the label for every metric
+- `ignoredRoutes` (optional)            - An array of routes to be exuded when calculating metrics. Default value: `['/metrics', '/healthz']`
+- `collectDefaultMetrics` (optional)    - A boolean indicating whether or not to collect [default nodejs metrics](https://github.com/siimon/prom-client/#default-metrics) (default: false)
```

Comment:
Thought:
This API is different from `ignoredRoutes`, we could expose a list of available metrics, and set this to a default value. This way, like with routes, we allow the users more granularity (for example if debugging some GC issues, not minding the eventloop).